editor/blocks/text-block/index.js

```diff
@@ -1,11 +1,11 @@
-const { query, html } = wp.blocks.query;
+const { html } = wp.blocks.query;
 
 wp.blocks.registerBlock( 'wp/text', {
 	title: 'Text',
 	icon: 'text',
 
 	attributes: {
-		value: query( 'p', html() )
+		value: html()
```

Comment:
I think this is unrelated to `wpautop`.

The `wpautop` needs to be implemented by catching the `Enter` key press and checking whether we should create a new block after the current one or not. (Similar behaviour can be seen on the `per-block`  prototype)